testsuite/tests/ui/mail_and_messages/test_email_application.py

```diff
@@ -0,0 +1,48 @@
+"""Test of automatic mails functionality connected to applications in UI"""
+import pytest
+
+from threescale_api.resources import Service
+from testsuite.ui.views.admin.audience.application import ApplicationDetailView
+
+pytestmark = pytest.mark.usefixtures("login")
+
+
+@pytest.fixture(scope="module")
+def app_id_key_service(service, navigator, application):
+    """Product with APP ID +APP KEY authentication"""
+    service.update({"backend_version": Service.AUTH_APP_ID_KEY})
+    service.proxy.list().update()
```

Comment:
wouldnt it be easier if added this in service_settings directly and didn't have to do a proxy update?